how-to/operations/enable-and-disable-maintenance-mode.rst

```diff
@@ -0,0 +1,51 @@
+Enable and Disable Maintenance Mode
+===================================
+
+Overview
+--------
+
+Maintenance mode helps protect the cluster during operations that could be disruptive. It is particularly useful when an operator needs to manually perform tasks on a node that may pose a risk of failure or data loss.
+
+Before proceeding, refer to the :doc:`Maintenance Mode </explanation/maintenance-mode>` to understand its functionality and impact.
+
+Deploy OpenStack Watcher Before Proceeding
+------------------------------------------
+
+Maintenance mode relies on `OpenStack Watcher`_ to manage hypervisor services and virtual machine instances. If the target node has an active compute role, ensure that Watcher is enabled before proceeding:
+
+.. code:: text
+
+   sunbeam enable resource-optimization
+
+Enabling Maintenance Mode
+-------------------------
+
+Before enabling maintenance mode, perform a dry run to check for potential issues:
```

Comment:
(question) is dry-run also do the **checking** or just print out the plan?

Reply:
dry-run do both the checks and print out the plan.